runtimes/java-11.0/src/main/java/io/openruntimes/java/Server.java

```diff
@@ -19,10 +19,10 @@ public static void main(String[] args) {
 
     public static Resp execute(Req req, Resp resp) {
         if (!req.headers().containsKey("x-internal-challenge") || req.headers().get("x-internal-challenge").isEmpty()) {
-            return resp.code(500).result("Unauthorized");
+            return resp.code(401).result("Unauthorized");
         }
         if (!req.headers().get("x-internal-challenge").equals(System.getenv("INTERNAL_RUNTIME_KEY"))) {
-            return resp.code(500).result("Unauthorized");
+            return resp.code(401).result("Unauthorized");
```

Comment:
This is a good suggestion, but, for now, we would like to keep this as is (500) to keep all the runtimes consistent

Reply:
Oh, whoops! I just saw 401 mentioned in the add-runtime.md and didn't check the other runtimes as reference; will revert this when I am back home!